Makefile

```diff
@@ -85,90 +85,90 @@ LIBC_TOP_HALF_MUSL_SRC_DIR = $(LIBC_TOP_HALF_MUSL_DIR)/src
 LIBC_TOP_HALF_MUSL_INC = $(LIBC_TOP_HALF_MUSL_DIR)/include
 LIBC_TOP_HALF_MUSL_SOURCES = \
     $(addprefix $(LIBC_TOP_HALF_MUSL_SRC_DIR)/, \
-        misc/a64l.c \
```

Comment:
this file still seems to contain a ton of whitespace changes.

Reply:
Sorry, I thought I had fixed that; will try again.